motr/m0crate/crate_index.c

```diff
@@ -923,6 +1071,11 @@ static int cr_idx_w_execute(struct cr_idx_w *w,
 	struct m0_fid		*keys = NULL;
 	struct kv_pair		 kv = {0};
 	int			 i;
+	/* key will contain <const value:random or sequential generated value> */
+	int kpart_one_size 	    = w->wit->key_size - w->wit->min_key_size;
+	char kpart_one[kpart_one_size];
+	m0_time_t             op_start_time;
+	m0_time_t             op_time;
```

Comment:
It is good to have all the variable names aligned. But they are not now.
You can just have a space between the type and variable name.

Reply:
Fixed.